README.md

```diff
@@ -98,6 +98,15 @@ or
 ```bash
 $ yarn test
 ```
+To run the spectron UI tests in this repository use:
+
+```bash
+$ yarn run spectron-test
```

Comment:
`run` is optional for yarn, so `yarn spectron-test` is sufficient. These could also be changed in the appveyor and circleci config files.